tests/integration/debug_files/upload.rs

```diff
@@ -0,0 +1,68 @@
+use crate::integration::{mock_endpoint, register_test, EndpointOptions};
+
+// NOTE: I have no idea why this is timing out on Windows.
+// I verified it manually, and this command works just fine. — Kamil
+#[cfg(not(windows))]
+#[test]
+fn command_debug_files_upload_help() {
+    register_test("debug_files/debug_files-upload-help.trycmd");
```

Comment:
windows CI is notoriously slow, esp for I/O operations. We ran into this when trying to run the Node CI in JS on windows: https://github.com/getsentry/sentry-javascript/pull/4616. Maybe we leave a TODO comment on the command as a future performance optimization?

Even opening a GH issue with the list of these disabled tests with `help wanted` might be nice, maybe somebody else can help figure it out for us!

Reply:
Updated